evaluations/php-rpc-lib_2_keeganquigley.md

```diff
@@ -0,0 +1,60 @@
+# Evaluation
+
+- **Status:** Accepted
+- **Application Document:** https://github.com/w3f/Grants-Program/blob/master/applications/php-rpc-lib.md
+- **Milestone:** 2
+- **Previously successfully merged evaluation:** Alxs, Keeganquigley
```

Comment:
```suggestion
- **Previously successfully merged evaluation:** All by Keeganquigley
```

Reply:
FYI - here we refer to evaluations that the author (in this case you) has already conducted and was merged.